conf/distro/poky-sota-nonostree.conf

```diff
@@ -0,0 +1,19 @@
+require conf/distro/poky.conf
+
+DISTRO = "poky-sota-nonostree"
+DISTRO_NAME = "OTA-enabled Linux"
+DISTRO_VERSION = "1.0"
+DISTRO_CODENAME = "sota-non-ostree"
+
+DISTRO_FEATURES_append = " systemd"
+VIRTUAL-RUNTIME_init_manager = "systemd"
+PREFERRED_RPROVIDER_virtual/network-configuration ??= "networkd-dhcp-conf"
+
+SOTA_HARDWARE_ID ??= "${MACHINE}-non-ostree"
```

Comment:
I am not sure that the non-ostree configuration requires another distro configuration as it actually does not use/add any 'distribution' as Yocto defines (it https://www.yoctoproject.org/docs/current/mega-manual/mega-manual.html#ref-features-distro).
Therefore, perhaps, it makes sense just simply move the nonostree specific stuff to the conf/local.conf.nonostree.append.

Reply:
The reason for such a decision is that the required configurations are placed in the distro config, and keep the local.conf clean, as much as possible.

Reply:
I think I'm still leaning towards just dumping this in local.conf. I don't think this belongs as an independent distro.